Example/Emission/ARRootViewController.m

```diff
@@ -253,60 +249,18 @@ - (ARCellData *)jumpToArtwork
   }];
 }
 
-- (ARCellData *)jumpToArtworkWithBNMO
+- (ARCellData *)jumpToPartner
 {
-  return [self tappableCellDataWithTitle:@"Artwork With BNMO" selection:^{
-    id viewController = [[ARArtworkComponentViewController alloc] initWithArtworkID:@"josh-reames-columbo-ii"];
+  return [self tappableCellDataWithTitle:@"Partner" selection:^{
+    id viewController = [[ARPartnerComponentViewController alloc] initWithPartnerID:@"dr-jorn-gunther-rare-books"];
     [self.navigationController pushViewController:viewController animated:YES];
   }];
 }
 
-- (ARCellData *)jumpToArtworkWithBNMOWithEdition
-{
-  return [self tappableCellDataWithTitle:@"Artwork With BNMO Edition Sets" selection:^{
-    id viewController = [[ARArtworkComponentViewController alloc] initWithArtworkID:@"terry-oneill-faye-dunaway-the-beverly-hills-hilton-los-angeles-the-morning-after-her-network-oscar-1"];
-    [self.navigationController pushViewController:viewController animated:YES];
-  }];
-}
-
-- (ARCellData *)jumpToArtworkWithInquireable
-{
-  return [self tappableCellDataWithTitle:@"Artwork With Inquireable" selection:^{
-    id viewController = [[ARArtworkComponentViewController alloc] initWithArtworkID:@"ryan-hewett-la-femme-iv"];
-    [self.navigationController pushViewController:viewController animated:YES];
-  }];
-}
-
-- (ARCellData *)jumpToArtworkInAuction
-{
-  return [self tappableCellDataWithTitle:@"Artwork In Auction" selection:^{
-    id viewController = [[ARArtworkComponentViewController alloc] initWithArtworkID:@"lucha-rodriguez-untitled"];
-    [self.navigationController pushViewController:viewController animated:YES];
-  }];
-}
-
-
-- (ARCellData *)jumpToArtworkWithMultipleImages
-{
-  return [self tappableCellDataWithTitle:@"Artwork With Multiple Images" selection:^{
-    id viewController = [[ARArtworkComponentViewController alloc] initWithArtworkID:@"francis-bacon-study-after-velazquez"];
-    [self.navigationController pushViewController:viewController animated:YES];
-  }];
-}
-
-- (ARCellData *)jumpToArtworkWithInstitutionPartner
-{
-  return [self tappableCellDataWithTitle:@"Artwork With Institution Partner" selection:^{
-    id viewController = [[ARArtworkComponentViewController alloc] initWithArtworkID:@"jean-leon-gerome-the-guard"];
-    [self.navigationController pushViewController:viewController animated:YES];
-  }];
-}
-
-
-- (ARCellData *)jumpToArtworkAttributionClassFAQ
+- (ARCellData *)jumpToPartner2
```

Comment:
I take it these changes were for local development? 😂

Reply:
Yes, I want to have more than one partner example while developing.